charts/workflow/requirements.yaml

```diff
@@ -45,6 +45,7 @@ dependencies:
     version: <registry-token-refresher-tag>
     repository: https://charts.deis.com/registry-token-refresher
   - name: router
+    condition: !global.experimental_native_ingress
```

Comment:
@mboersma I don't like the "big {{ if }}" approach because it requires changes to the router chart that endow that chart with an awareness that some parent chart might include it but wish for it to do nothing. That doesn't seem very clean to me.